lib/bookingsync_portal.rb

```diff
@@ -21,6 +21,13 @@ module BookingsyncPortal
   mattr_accessor :portal_name
   @@portal_name = 'Portal'
 
+  # source name for use in bookings
+  mattr_accessor :source_name
+
+  # Allow to create a remote rental from this app.
+  mattr_accessor :create_remote_rental
+  @@create_remote_rental = false
```

Comment:
Yeah, though so, just saying that class instance variable works in most cases exactly the same as class variable, but is safer to use.